extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQControllerTask.java

```diff
@@ -204,6 +204,12 @@ public void stopGracefully(final TaskConfig taskConfig)
     }
   }
 
+  @Override
+  public int getPriority()
+  {
+    return getContextValue(Tasks.PRIORITY_KEY, Tasks.DEFAULT_BATCH_INDEX_TASK_PRIORITY);
```

Comment:
Nit: should we document this here https://druid.apache.org/docs/24.0.0/ingestion/tasks.html#lock-priority ?

Reply:
👍  updated.